packages/apollo/README.md

```diff
@@ -455,3 +455,6 @@ It can also be helpful to print standard out during testing. To enable logging,
 ```
 .stdout({ print: true })
 ```
+
+## Publishing
```

Comment:
Can we match the existing formatting in this file and put a newline after the heading?